src/main/pages/user-guide/creating-starting-workspaces.md

```diff
@@ -29,6 +29,33 @@ In this scenario, you need to author a custom [recipe][recipes], then a stack, a
 
 {% include image.html file="workspaces/custom_recipe_stack.gif" %}
 
+**Stack Sharing**
```

Comment:
Should we expend the section to be "Configure System Stacks". Where we explain that an admin can edit/remove the stacks available to all users ?